src/components/recs/rec-tile.tsx

```diff
@@ -3,29 +3,150 @@
 import RecTitle from "./rec-title";
 import RecMap from "./rec-map";
 import RecFooter from "./rec-footer";
-import { useContext } from "react";
+import { useContext, useState } from "react";
 import { WindowContext } from "../provider/window-provider";
 import { IRecordedGame } from "@/types/RecordedGame";
 import { useTeams } from "@/hooks/useTeams";
+import { SquarePen, Trash2 } from "lucide-react";
+import { Sheet, SheetContent, SheetDescription, SheetHeader, SheetTitle, SheetTrigger } from "../ui/sheet";
+import { Button } from "../ui/button";
+import { toast } from "../ui/use-toast";
 
 interface RecTileProps {
+  id:string
```

Comment:
Are you using prettier? It should be adding a space here...maybe we need to config it.

Reply:
Yes, i'm using prettier